onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -234,28 +234,29 @@ void AddTensorAsPyObj(OrtValue& val, std::vector<py::object>& pyobjs) {
   GetPyObjFromTensor(rtensor, obj);
   pyobjs.push_back(obj);
 }
-
 class SessionObjectInitializer {
  public:
   typedef const SessionOptions& Arg1;
-  typedef logging::LoggingManager* Arg2;
+  // typedef logging::LoggingManager* Arg2;
+  static std::string default_logger_id;
   operator Arg1() {
```

Comment:
> operator Arg1() { [](start = 1, length = 18)

I would not do it, otherwise would add const qualifier

Reply:
The Arg1() thing existed before. 
Is your comment about the default_logger_id?